src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -838,3 +838,11 @@ def load_arguments(self, _):
         c.argument('analytics_query', options_list=['--analytics-query', '-q'], help="Query to execute over Log Analytics data.")
         c.argument('timespan', help="Timespan over which to query. Defaults to querying all available data.")
     # endregion
+
+    # region disk encryption set
+    with self.argument_context('disk-encryption-set create') as c:
+        c.argument('disk_encryption_set_name', name_arg_type, help='Name of disk encryption set.')
+        c.argument('key_url', help='URL pointing to a key or secret in KeyVault.')
```

Comment:
is the key_url same with Swagger property name? it's confusing with help message.

Reply:
> 
> 
> is the key_url same with Swagger property name? it's confusing with help message.

Yes.
e.g. https://vault4848.vault.azure.net/keys/key1/e3a5735dfa4d47b8879fa61cdd1cfb82

https://github.com/Azure/azure-rest-api-specs/blob/master/specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/disk.json#L1432

Reply:
Maybe the name `--key` or `--active-key` is better than `--key-url`.